planetwars/client/app/actions/matches/matches.ts

```diff
@@ -140,39 +141,44 @@ export function runMatch() {
       };
     });
 
-    const config: PwClient.MatchParams = {
+    const config: PwClient.ServerParams = {
       address: params.address,
       logFile: match.logPath,
       ctrl_token: params.ctrl_token,
     };
 
     console.log("This probably doesn't work!!!!");
 
-    PwClient.MatchRunner.create(Config.matchRunner, config).then((runner) => {
```

Comment:
Is this no longer a Promise? (don't have a real setup here to easily check myself)